### ISSUE TITLE
installing exodus redux

### ISSUE THREAD

**Issue by xmanrick** [I-A-C/I-A-C.github.io#20]
when installing exodus redux i got (to repository install) and i got a message "could not connect to repository" (using nexus 20) how can i fix it?

**Comment by kwag** [comment]
Same here. Seems this project is dead.

**Comment by aamirmukaram** [comment]
same here

**Comment by protected22** [comment]
Anyone found an alternative?

**Comment by xmanrick** [comment]
You have to back and install kodi 18 or when it was out. It doesn't work
with anything else.

**Comment by eeden003** [comment]
> when installing exodus redux i got (to repository install) and i got a message "could not connect to repository" (using nexus 20) how can i fix it?

s

**Comment by xmanrick** [comment]
You CAN'T use nexus 20 or anything other than kodi 18

**Comment by protected22** [comment]
Got it working on Kodi 18. But looks like there are no streams available anymore through Exodus Redux. I tried multiple TV Shows and movies and all failed saying "No streams available"

**Comment by eeden003** [comment]
Yes! Exodus Redux searches on old sites that have been moved or deleted. No scraper-updates since years and years, so....it cant find the current sites !
